wbooks/src/components/comment/strings.js

```diff
@@ -0,0 +1,8 @@
+const strings = {
```

Comment:
by convention, strings its a file that contain a list of constants:
```
export const STRING1 = 'firstString';
export const ANOTHER_STRING = 'this is another string'
```
Using mayus and snake_case for constants.
then, you import strings file on the view like
```
import { STRING1, ANOTHER_STRING } from './strings';
```